scripts/system/create/entityProperties/html/entityProperties.html

```diff
@@ -1,4 +1,11 @@
 <!--
+//VERSION 2.0
+//  Modified by Alezia Kurdis on on 02/27/2020
+//  for "Project Athena"
+//
+//  Addition of a tab mechanism instead of collapsible sections to reduce the scrolling.
+//
+//VERSION 1.0
```

Comment:
I think that "versions" and a list of changes made are not a good idea:
- Versions implies that the version of the script is managed independently of the Athena releases.
- Listing changes implies that all changes in the script are and will continue to be listed.
(The "Modified by David Back on 19 Oct 2018" line is an unusual outlier; such "modified by" lines were normally not included in High Fidelity scripts.)

Instead, just adding a second copyright line suffices:
```
//  Copyright 2020 Project Athena contributors.
```
[See section 4.3.4 in the Coding Standard.]